tests/unit/objectives/test_single_objectives.py

```diff
@@ -50,6 +59,9 @@
     hartmann_3,
     hartmann_6,
     logarithmic_goldstein_price,
+    michalewicz_2d,
```

Comment:
we should probably be consistent with naming of other functions and remove `d` part at the end in these 2d, 5d and 10d

Reply:
Good point, I've fixed this now. Anything else?